core/src/main/kotlin/org/evomaster/core/output/oracles/ImplementedOracle.kt

```diff
@@ -75,9 +75,13 @@ abstract class ImplementedOracle {
      * This is a workaround (ScoutAPI only sees paths without the prefix, others with the prefix).
      * Longer term, this could also be a place to handle any additional peculiarities with SUT specific
      * OpenAPI standards.
+     *
+     * The same applies where the prefix is "v2" (e.g. language tools).
      */
+
     fun retrievePath(objectGenerator: ObjectGenerator, call: RestCallAction): PathItem? {
         return objectGenerator.getSwagger().paths.get(call.path.toString()) ?:
-        objectGenerator.getSwagger().paths.get(call.path.toString().removePrefix("/api"))
+        objectGenerator.getSwagger().paths.get(call.path.toString().removePrefix("/api")) ?:
+        objectGenerator.getSwagger().paths.get(call.path.toString().removePrefix("/v2"))
```

Comment:
I m a bit skeptical about these adhoc hacks.
the handling of "basePath" has changed between V2 and V3 versions of OpenApi, but it is still info available from the schema, eg, see addActionsFromSwagger